src/Native/Runtime/unix/PalRedhawkInline.h

```diff
@@ -90,7 +90,7 @@ FORCEINLINE void PalMemoryBarrier()
     __sync_synchronize();
 }
 
-#define PalDebugBreak() __builtin_trap()
+#define PalDebugBreak() abort()
```

Comment:
This is because if __bultin_trap() is used with illegal instruction the gdb shows the backtrace incorrectly. Abort or assert shows a valid bt.